### ISSUE TITLE
Idea: A dll "proxy" to prevent wrong installations

### ISSUE THREAD

**Issue by AlexejheroYTB** [QModManager/QModManager#104]
With the engine update to 2018, you can guarantee that there will be users trying to install QModManager 3.0 for cracked versions not yet on U2018.

I suggest turning `QModInstaller.dll` intro a proxy file which checks for the unity version, nitrox, and does all initializing before `QModManager.dll` is loaded into the app domain.

**Comment by AlexejheroYTB** [comment]
To do - after #81

**Comment by AlexejheroYTB** [comment]
Merged into #98
